vcd/provider.go

```diff
@@ -95,10 +96,10 @@ func Provider() terraform.ResourceProvider {
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	maxRetryTimeout := d.Get("max_retry_timeout").(int)
 
-	// TODO: Deprecated, remove in next major release
-	// if v, ok := d.GetOk("maxRetryTimeout"); ok {
-	// 	maxRetryTimeout = v.(int)
-	// }
+	// It was deprecated, and now removed
+	//if v, ok := d.GetOk("maxRetryTimeout"); ok {
+	//	maxRetryTimeout = v.(int)
+	//}
 
```

Comment:
Could you elaborate here what happen?

Reply:
A messed up merge, I assume. Will clean it up.